og_format_examples_files/sg558_20240206T000000_R.cdl

```diff
@@ -0,0 +1,549 @@
+		INSTRUMENT_DATA\ LOGGERS_sg558:maker = "Kongsberg Maritime" ;
+		INSTRUMENT_DATA\ LOGGERS_sg558:maker_vocabulary = "http://vocab.nerc.ac.uk/collection/L35/current/MAN0015/" ;
+		INSTRUMENT_DATA\ LOGGERS_sg558:model = "Seaglider M1 Glider data logger" ;
+		INSTRUMENT_DATA\ LOGGERS_sg558:model_vocabulary = "https://vocab.nerc.ac.uk/collection/L22/current/TOOL1185" ;
+		INSTRUMENT_DATA\ LOGGERS_sg558:serial_number = "sg558" ;
+		INSTRUMENT_DATA\ LOGGERS_sg558:calibration_date = "" ;
+	string TRAJECTORY ;
+		TRAJECTORY:cf_role = "trajectory_id" ;
+		TRAJECTORY:long_name = "trajectory name" ;
+		TRAJECTORY:data_mode_vocabulary = "" ;
+	string PLATFORM_TYPE ;
+		PLATFORM_TYPE:long_name = "type of glider" ;
+		PLATFORM_TYPE:platform_type_vocabulary = "" ;
+	string PLATFORM_MODEL ;
+		PLATFORM_MODEL:long_name = "model of the glider" ;
+		PLATFORM_MODEL:platform_model_vocabulary = "" ;
+	string WMO_IDENTIFIER ;
+		WMO_IDENTIFIER:long_name = "wmo id" ;
+	string DEPLOYMENT_TIME ;
+		DEPLOYMENT_TIME:long_name = "Date of deployment" ;
+		DEPLOYMENT_TIME:calendar = "gregorian" ;
+		DEPLOYMENT_TIME:standard_name = "time" ;
+		DEPLOYMENT_TIME:units = "seconds since 1970-01-01T00:00:00Z" ;
+		DEPLOYMENT_TIME:axis = "T" ;
+	string DEPLOYMENT_LATITUDE ;
+		DEPLOYMENT_LATITUDE:long_name = "latitude of deployment" ;
+	string DEPLOYMENT_LONGITUDE ;
+		DEPLOYMENT_LONGITUDE:long_name = "longitude of deployment" ;
+
+// global attributes:
+		:geospatial_bounds_crs = "EPSG:4326" ;
+		:geospatial_bounds_vertical_crs = "EPSG:5831" ;
+		:geospatial_lat_min = -63.72288f ;
+		:geospatial_lat_max = -63.34504f ;
+		:geospatial_lon_min = -53.3449f ;
+		:geospatial_lon_max = -51.54048f ;
+		:geospatial_bounds = "POLYGON ((-53.34490203857422, -51.54047775268555, -63.72288131713867, -63.34503936767578))" ;
+		:geospatial_vertical_min = -0.1495176f ;
+		:geospatial_vertical_max = 751.8447f ;
+		:geospatial_vertical_positive = "down" ;
+		:geospatial_vertical_units = "m" ;
+		:time_coverage_start = "2024-02-06T17:05Z" ;
+		:time_coverage_end = "2024-02-23T20:50Z" ;
+		:agency = "UEA" ;
+		:agency_role = "contact point" ;
+		:agency_role_vocabulary = "https://edmo.seadatanet.org/" ;
+		:agency_id = "" ;
+		:agency_id_vocabulary = "EDMO" ;
+		:contributor_name = "Karen Heywood" ;
+		:contributor_email = "k.heywood@uea.ac.uk" ;
+		:contributor_role = "Principal investigator" ;
+		:contributor_role_vocabulary = "BODC person database" ;
+		:contributor_id = "" ;
+		:publisher_name = "NOC" ;
+		:publisher_email = "glidersbodc@bodc.ac.uk" ;
+		:publisher_url = "https://www.bodc.ac.uk" ;
+		:publisher_type = "DAC" ;
+		:publisher_institution = "NOC" ;
+		:creator_name = "NOC" ;
+		:creator_email = "glidersbodc@bodc.ac.uk" ;
+		:creator_url = "https://www.noc.ac.uk" ;
+		:creator_type = "DAC" ;
+		:creator_institution = "NOC" ;
+		:wmoid = "8901054" ;
+		:institution = "" ;
+		:comment = "" ;
+		:history = "" ;
+		:keywords = "" ;
+		:doi = "" ;
+		:ctd_make_model = "" ;
```

Comment:
quite a few empty attrs that do not feature in the OGformat, like ctd_make_model. Consider removing them?

Reply:
Yep will remove these legacy attrs. doi i believe is still in the manual as a global attribute - probably should keep this even if empty.